test/views/components/RenderTextSpec.scala

```diff
@@ -36,7 +36,7 @@ class RenderTextSpec extends WordSpec with Matchers with GuiceOneAppPerSuite {
 
   trait Test {
     implicit val labels: Labels = LabelCache(Map("Blah" -> Label("Blah", Some("a value")),
-                                                                "A-Label" -> Label("A-Label", Some("33.99"))))
+                                                                "A-Label" -> Label("A-Label", Some("33.9"))))
```

Comment:
Could you add a label that has a value >= 1000 to test commas and an exact number to test that .00 added for currency, then add a couple of tests

Reply:
yes, will do